tools/rpm2img

```diff
@@ -383,22 +392,28 @@ cp /host/tools/bootconfig/empty-bootconfig.data "${PRIVATE_MOUNT}/bootconfig.dat
 mkfs.ext4 -b 4096 -i 4096 -I 256 -d "${PRIVATE_MOUNT}" "${PRIVATE_IMAGE}" "${partsize[PRIVATE]}M"
 dd if="${PRIVATE_IMAGE}" of="${OS_IMAGE}" conv=notrunc bs=1M seek="${partoff[PRIVATE]}"
 
-# BOTTLEROCKET-DATA
+# BOTTLEROCKET-DATA-A and BOTTLEROCKET-DATA-B
 
 # If we build on a host with SELinux enabled, we could end up with labels that
 # do not match our policy. Since we allow replacing the data volume at runtime,
 # we can't count on these labels being correct in any case, and it's better to
 # remove them all.
 UNLABELED=$(find "${DATA_MOUNT}" \
     | awk -v root="${DATA_MOUNT}" '{gsub(root"/","/"); gsub(root,"/"); print "ea_rm", $1, "security.selinux"}')
-mkfs.ext4 -d "${DATA_MOUNT}" "${BOTTLEROCKET_DATA}" "${partsize[DATA]}M"
-echo "${UNLABELED}" | debugfs -w -f - "${BOTTLEROCKET_DATA}"
+mkfs.ext4 -d "${DATA_MOUNT}" "${BOTTLEROCKET_DATA_A}" "${partsize[DATA-A]}M"
+echo "${UNLABELED}" | debugfs -w -f - "${BOTTLEROCKET_DATA_A}"
```

Comment:
This changes the scope a bit but this might actually be a good time to try an optimization idea that's come up recently - creating the ext4 filesystem on the running system. That will help the align the initial filesystem parameters with the defaults for the actual size of the filesystem, rather than for an empty directory.

To do that, we'd drop all of the `mkfs.ext4` / `debugfs` logic here and create a new `prepare-local-fs.service` that `local.mount` would have Requires / After dependencies on. Something like this:

```
[Unit]
Description=Prepare Local Filesystem (/local)
DefaultDependencies=no
Wants=dev-disk-by\x2dpartlabel-BOTTLEROCKET\x2dDATA.device
After=dev-disk-by\x2dpartlabel-BOTTLEROCKET\x2dDATA.device
RefuseManualStart=true
RefuseManualStop=true

[Service]
Type=oneshot

# Resize the partition, whether or not it resides on the same disk as /.
ExecStart=/usr/bin/systemd-repart --dry-run=no /dev/disk/by-partlabel/BOTTLEROCKET-DATA

# Create the filesystem on the partition, if it doesn't exist.
ExecStart=/usr/lib/systemd/systemd-makefs ext4 /dev/disk/by-partlabel/BOTTLEROCKET-DATA

RemainAfterExit=true
StandardError=journal+console

# systemd-repart returns 77 if there's no existing GPT partition table
SuccessExitStatus=77

[Install]
WantedBy=local-fs.target
```

(`repart-local.service` would also change slightly, to drop the `systemd-repart` and `SuccessExitStatus`.)